### ISSUE TITLE
GA: real-time reporting in javascript?

### ISSUE THREAD

**Issue by dhousman** [FreeUKGen/FreeCENMigration#671]
it's possible to create real-time reports to be embedded on web pages. Create reports of ___ metrics in real time.

**Comment by Captainkirkdawson** [comment]
Reports of what and for what purpose and why javascript as the solution

**Comment by richpomfret** [comment]
We need to first define the requirements here and how it may be useful. We can look at this in terms of the GA site behaviours and then project database data visualisation.

**Comment by dhousman** [comment]
@richpomfret yes this could be useful in embedding a report that publicly shares  information about how much the site is currently being used, or possibly to visualize site goals such as conversion (possibly transcribing data). Note sure if you want/need something like this, but I wanted to let you know that's an option.

**Comment by richpomfret** [comment]
Thanks @dhousman. I think we will put this on the 'back-burner' for now, seeing as we do not have a clear set of requirements for it.
